res/css/views/dialogs/_RegistrationEmailPromptDialog.scss

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_RegistrationEmailPromptDialog {
+    width: 417px;
```

Comment:
It is the width design asked for, and it is different to the other magic width because

> it was, but the precise widths weren't a strong decision - would be ok if they're the same width.  i was loosely thinking the people who go ahead without email are more likely to be less put off by a wider modal, a smaller one feels less effort.  while people who interact with the homeserver are more likely to be invested in the experience.  not sure if we have small, medium, large modals as concepts.  I didn't think this through much beyond that 😕